tests/conftest.py

```diff
@@ -11,7 +11,7 @@
 @pytest.fixture
 def config():
     return {
-        "user": os.getenv("USER"),
+        "user": os.getenv("XACTLY_USER"),
```

Comment:
Good catch on this. I didn't realize `USER` was a Mac environment variable, and it would explain why I didn't run into that issue locally.